### PR TITLE
Collaboration guidelines - reorg and simplification

### PR DESCRIPTION
**Pull request by krisdale**
- Reorg content into 3 pages for collab guides: overview (collaboration.rst),
structure and formatting, and writing guide. (includes removal of old pages)
- Simplify content, including removal of definitions/guidelines provided by
our listed resources, simplification of wording and examples.
- Add in quick ref section for our updated BKMs
- Update the README installation instructions
- Clarify different methods of contribution

### CODE REVIEW

**Comment by krisdale** [comment]
@MCamp859 @mvincerx @intelkevinputnam - overhaul of collaboration guidelines. Please review and provide feedback as needed. Mary has already provided the initial content review. Thanks!

File: README.rst [hunk 1]
```diff
@@ -1,83 +1,74 @@
...
-
-.. code-block:: console
-
-   $ sudo apt-get install python-pip
-   $ sudo pip install -U sphinx sphinx-autobuild
+.. todo add comment re not using standards here.
 
+`Clear Linux\* OS documentation`_ is written using `reStructuredText`_ and
+built using `Sphinx`_. Documentation may be built locally for development and
+testing purposes by following the instructions in this README.
```
Comment (MCamp859) [inline README.rst]: Suggest reordering sentence:  Follow the instructions in this README to build the documentation locally for development and testing.

File: README.rst [hunk 2]
```diff
@@ -1,83 +1,74 @@
...
 
-Run make
-========
+Run the build
+*************
 
-Finally are we ready to run :command:`make`. Be sure to :command:`cd` to the
-:file:`source/` directory where your ``.rst`` files are, before
-running :command:`make html`, or the doc format of your choice.
+We build our documentation using . In the source directory of your
```
Comment (MCamp859) [inline README.rst]: Change: We build our documentation using Sphinx.

File: source/clear-linux/reference/collaboration/structure-formatting.rst [hunk 3]
```diff
@@ -0,0 +1,483 @@
...
+clarity.
+
+* All files must have a top level heading, which is the title for the page.
+* Headings should be short and descriptive.
+* Write headings in sentence case (capitalize only the first word and proper
+  nouns).
+* Up to three additional levels of headings are allowed under the title heading.
+* Each heading must be followed by at least one paragraph of content. There
+  should never be two consecutive headings.
+
```
Comment (MCamp859) [inline source/clear-linux/reference/collaboration/structure-formatting.rst]: Add an example:  
**Use this:**
Install software

**Not this:**
Installing the Software on the Platform

File: source/clear-linux/reference/collaboration/structure-formatting.rst [hunk 4]
```diff
@@ -0,0 +1,483 @@
...
+Headings
+========
+
+Use headings to section and organize your content for better readability and
+clarity.
+
+* All files must have a top level heading, which is the title for the page.
+* Headings should be short and descriptive.
+* Write headings in sentence case (capitalize only the first word and proper
```
Comment (MCamp859) [inline source/clear-linux/reference/collaboration/structure-formatting.rst]: Add new bullet:
* Use strong verbs in headings, but not -ing verbs, such as Running, Testing, etc.

File: README.rst (hunk 1 above)
Comment (krisdale) [reply README.rst]: Done.

File: README.rst (hunk 2 above)
Comment (krisdale) [reply README.rst]: Done

File: source/clear-linux/reference/collaboration/structure-formatting.rst (hunk 3 above)
Comment (krisdale) [reply source/clear-linux/reference/collaboration/structure-formatting.rst]: Great example. I added this example and moved the grammar/style related bullets into the writing guide section on headings. (trying to keep style/grammar in one doc, structure/formatting in the other).

File: source/clear-linux/reference/collaboration/structure-formatting.rst (hunk 4 above)
Comment (krisdale) [reply source/clear-linux/reference/collaboration/structure-formatting.rst]: Yes, done!

**Comment by krisdale** [comment]
@MCamp859 - updated this PR with your feedback. Thanks!

File: source/clear-linux/reference/collaboration/structure-formatting.rst (hunk 3 above)
Comment (MCamp859) [reply source/clear-linux/reference/collaboration/structure-formatting.rst]: Your rewrites are even better - LGTM.

File: source/clear-linux/reference/collaboration/writing-guide.rst [hunk 5]
```diff
@@ -0,0 +1,419 @@
...
+Use short sentences and paragraphs. Stick to the principle of one main
+idea per sentence, plus one additional point if needed. Each paragraph
+should address one main idea. Remember the basic structure of a paragraph:
+Introduction, body, and conclusion.
+
+Be friendly
+===========
+
+We write for our peers and want to be familiar. Use the second person, you or
+we. Be professional, respectful, and cooperative.
```
Comment (intelkevinputnam) [inline source/clear-linux/reference/collaboration/writing-guide.rst]: "we" is 1st person plural. Maybe just "Use you or we."

File: source/clear-linux/reference/collaboration/writing-guide.rst [hunk 6]
```diff
@@ -0,0 +1,419 @@
...
+
+**Not this:** ::
+
+  Power management mechanism integration policies.
+
+Or:
+
+**Use this:** ::
+
+  Requirements for test desks that measure signal integrity.
```
Comment (intelkevinputnam) [inline source/clear-linux/reference/collaboration/writing-guide.rst]: "test desks" here and "test deck" below.

File: source/clear-linux/reference/collaboration/structure-formatting.rst [hunk 7]
```diff
@@ -0,0 +1,483 @@
...
+Our documentation is written in the reStructuredText markup language, using
+Sphinx roles and directives. We use Sphinx to generate the final documentation.
+You can read more about reStructuredText and Sphinx on their respective
+websites:
+
+* `Sphinx documentation`_
+* `reStructuredText Primer`_
+
+You can view the content directly in the .rst markup files, or you can generate
+the HTML content by installing and running the documentation locally. To run the
```
Comment (intelkevinputnam) [inline source/clear-linux/reference/collaboration/structure-formatting.rst]: "running the documentation locally" sounds funny to me. I'd prefer building.

File: source/clear-linux/reference/collaboration/structure-formatting.rst [hunk 8]
```diff
@@ -0,0 +1,483 @@
...
+
+Limit line length to 78 characters. The GitHub web interface forces this
+limitation for readability.
+
+Remove trailing whitespace from your documents.
+
+Code blocks and examples
+************************
+
+When providing example code or commands that are less than 10 lines, use the
```
Comment (intelkevinputnam) [inline source/clear-linux/reference/collaboration/structure-formatting.rst]: Is there a reason that this is limited to 10 lines? What do we do if it needs to be longer?

**Comment by mvincerx** [comment]
I'll leave this PR to you all. I appreciate how everyone has collaborated on this to make it the best it can be.  Just let me know when you're ready for a merge. 👍

File: source/clear-linux/reference/collaboration/writing-guide.rst (hunk 5 above)
Comment (krisdale) [reply source/clear-linux/reference/collaboration/writing-guide.rst]: Done.

File: source/clear-linux/reference/collaboration/writing-guide.rst (hunk 6 above)
Comment (krisdale) [reply source/clear-linux/reference/collaboration/writing-guide.rst]: Removed second example as it wasn't necessary. :)

File: source/clear-linux/reference/collaboration/structure-formatting.rst (hunk 7 above)
Comment (krisdale) [reply source/clear-linux/reference/collaboration/structure-formatting.rst]: Done.

File: source/clear-linux/reference/collaboration/structure-formatting.rst (hunk 8 above)
Comment (krisdale) [reply source/clear-linux/reference/collaboration/structure-formatting.rst]: Removed, viva la example code!

**Comment by krisdale** [comment]
@intelkevinputnam Made edits based on your feedback. Please give it a quick check and comment if you think its ready to merge. Thanks!

**Comment by mvincerx** [comment]
Amazing work and collaboration on this PR. 
I especially like the section on "Inline text formatting," which spells out your BKMs!  LGTM. 

Reminder: Don't forget to rebase. Lots of big structural changes in past 2 days.

**Pull request merged**